README.md

```diff
@@ -0,0 +1,66 @@
+# Algebra In Lean
+
+This is the repository for Algebra In Lean (AIL), an interactive problem set
+that takes users through an advanced course in Lean through the lens of abstract
+algebra. The goal of this problem set is to introduce users to formalization
+and Lean 4 while also assisting in writing rigorous proofs in
+group theory. Beginning with introductions to logic-based reasoning in Lean, each chapter takes users through various topics in group theory. Akin to Kevin
```

Comment:
```suggestion
This is the repository for Algebra In Lean (AIL), an interactive problem set
that takes users through an advanced course in Lean through the lens of
abstract algebra. The goal of this problem set is to introduce users to
formalization and Lean 4 while also assisting in writing rigorous proofs in
group theory. Beginning with introductions to logic-based reasoning in Lean,
each chapter takes users through various topics in group theory. Akin to Kevin
Buzzard’s [Formalising
Mathematics](https://www.ma.imperial.ac.uk/~buzzard/xena/formalising-mathematics-2024)
at Imperial College London, AIL includes chapters on group axioms,
homomorphisms, and subgroups, each separated into problem sheets. These problem
sheets include both the formalization of mathematical structures and exercises
on proving related theorems.
```

Just to keep wrapping (at 80, I think?) consistent. Fix "Formalizing Mathematics course" to "Formalizing Mathematics". Fix "the Imperial College London" to "Imperial College London".